src/FakeItEasy/Expressions/CallExpressionParser.cs

```diff
@@ -61,7 +61,7 @@ private static ParsedCallExpression ParsePropertyCallExpression(MemberExpression
             }
 
             return new ParsedCallExpression(
-                calledMethod: property.GetGetMethod(true),
+                calledMethod: property.GetGetMethod(true)!,

```

Comment:
The property comes from a MemberExpression, and since an expression tree can't have an assignment, it has to be a read access, so there has to be a getter (unless, of course, the expression was created manually... should we handle this case?)

Reply:
Let's wait for the bug report!